### PR TITLE
Changes to make it work

### PR DESCRIPTION
**Pull request by mattfullerton**
I'm assuming the Docker Ubuntu image is independent of my Ubuntu (?). If not, it could be that some of these changes are only necessary for Ubuntu 14.04.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -23,10 +23,6 @@ RUN cd /var/www ; npm install
 # we want to run in production mode
 ENV NODE_ENV production
 
-# LaTeX 2012
```
Comment (aslushnikov) [inline Dockerfile]: Could you please clarify why did you remove this?

File: Dockerfile [hunk 2]
```diff
@@ -10,7 +10,7 @@ MAINTAINER Andrey Lushnikov aslushnikov@gmail.com
 # install node and memcached
 RUN echo "deb http://us.archive.ubuntu.com/ubuntu/ precise universe" >> /etc/apt/sources.list
 RUN apt-get update
-RUN apt-get install -y python-software-properties python python-setuptools ruby rubygems
+RUN apt-get install -y software-properties-common python-software-properties python python-setuptools ruby
```
Comment (aslushnikov) [inline Dockerfile]: What is "software-properties-common" for?

**Comment by aslushnikov** [comment]
Thank you for the PR, sorry for the long response. Could you please squash the commits into a single one with a good description?

File: Dockerfile (hunk 2 above)
Comment (mattfullerton) [inline Dockerfile]: To provide add-apt-repository

File: Dockerfile (hunk 1 above)
Comment (mattfullerton) [inline Dockerfile]: Sorry, there's no good reason. I'm a bit delete-button-happy sometimes.

**Comment by mattfullerton** [comment]
Closing, will be replaced with cleaner PR
